packages/vats/tools/fake-bridge.js

```diff
@@ -254,6 +254,9 @@ export const makeFakeTransferBridge = (zone, onToBridge = () => {}) => {
           registered.delete(params.target);
           return undefined;
         }
+        case 'IBC_METHOD': {
+          return undefined;
+        }
```

Comment:
It doesn't seem like we should add this here - `vtransfer/keeper/keeper.go` only knows about `BRIDGE_TARGET_(UN)REGISTER`.

Without this, our test runner prints some uncaught exceptions:

```bash
yarn run v1.22.22
$ ava test/exos/local-orchestration-account-kit.test.ts -m 'monitor transfers'

IBC onCreate
----- FakeBridge.5  2 toBridge VLOCALCHAIN_ALLOCATE_ADDRESS undefined {}
----- FakeBridge.5  3 toBridge BRIDGE_TARGET_REGISTER { target: 'agoric1fakeLCAAddress' }
----- FakeBridge.5  4 toBridge IBC_METHOD { method: 'receiveExecuted', packet: 'a packet', ack: undefined }
Error sending ack: (Error#1)
Error#1: unknown type IBC_METHOD
  at Object.toBridge (packages/vats/tools/fake-bridge.js:261:15)

----- FakeBridge.5  5 toBridge IBC_METHOD {
  method: 'receiveExecuted',
  packet: 'a packet',
  ack: 'eyJlcnJvciI6InVua25vd24gdHlwZSBcIklCQ19NRVRIT0RcIiJ9'
}
  ✔ monitor transfers
    ℹ bootstrap vat dependencies
    ℹ exo setup - prepareLocalChainAccountKit
    ℹ request account from vat-localchain
    ℹ make a LocalChainAccountKit
    ℹ receiveUpcall {
        event: 'writeAcknowledgement',
        packet: 'a packet',
        target: 'agoric1fakeLCAAddress',
        type: 'VTRANSFER_IBC_EVENT',
      }
Error in handling of {
  event: 'writeAcknowledgement',
  packet: 'a packet',
  target: 'agoric1fakeLCAAddress',
  type: 'VTRANSFER_IBC_EVENT'
} (Error#2)
Error#2: unknown type IBC_METHOD
  at Object.toBridge (packages/vats/tools/fake-bridge.js:261:15)

  ─

  1 test passed
✨  Done in 1.36s.
```

Reply:
This was figured out. I included a `VTransferIBCEvent` type with thoroughish comments and mocks for unit testing